packages/account-sdk/src/interface/public-utilities/prolink/createProlinkUrl.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2018-2025 Coinbase, Inc. <https://www.coinbase.com/>
+
+/**
+ * Base App universal link utilities for prolinks
+ */
+
+/**
+ * Create a link with an encoded prolink query parameter and additional query parameters
+ *
+ * @param prolink - Base64url-encoded prolink payload
+ * @param baseUrl - Base URL to use for the link, defaults to https://base.app/base-pay
+ * @param additionalQueryParams - Additional query parameters to add to the link
+ * @returns { link: string } - Object containing the full link
```

Comment:
preference for returning the string directly - let me know if you have thoughts in the other direction